package.json

```diff
@@ -70,17 +70,30 @@
     "prettier": "^2.1.2",
     "ts-node": "^9.1.1",
     "ttypescript": "^1.5.12",
-    "type-coverage": "^2.14.8",
+    "type-coverage": "^2.17.0",
     "typescript": "4.1.3"
   },
+  "engines": {
+    "node": ">=14.4.0",
+    "pnpm": ">=5"
```

Comment:
I think we should add npm: >=999 and yarn: >=999 to prevent people from using those tools

Reply:
new pr handle this problem?